DQMOffline/Lumi/BuildFile.xml

```diff
@@ -0,0 +1,9 @@
+<use name="FWCore/Framework"/>
+<use name="DataFormats/MuonReco"/>
+<use name="DataFormats/HLTReco"/>
+<use name="TrackingTools/IPTools"/>
+<use name="boost"/>
+<use name="root"/>
```

Comment:
you should not use all of the root scram tool - which classes are needed? probably rootcore is enough